test/HiGlassComponent/divergent-track.js

```diff
@@ -31,17 +31,13 @@ describe('Divergent tracks', () => {
     // await fetchMockHelper.storeDataAndResetFetchMock();
   });
 
-  it('Check that there are green and red rects', (done) => {
+  it('Check that there are green and red rects', async () => {
     const svg = hgc.instance().createSVG();
-
-    const svgText = new XMLSerializer().serializeToString(svg);
     expect(
-      svgText.indexOf('fill="green" stroke="green" x="11.24963759567723"'),
-    ).to.be.greaterThan(0);
+      svg.querySelector("rect[fill='green'][stroke='green'][x^='11.24963759567']")
```

Comment:
the `x^=val` is a [substring match](https://www.w3.org/TR/selectors/#attribute-substrings) rather than exact match, allowing us to leave off some significant digits which were causing issues with the test passing.